sockeye/rnn.py

```diff
@@ -48,20 +48,54 @@ def __init__(self,
         self.forget_bias = forget_bias
 
 
-def get_stacked_rnn(config: RNNConfig, prefix: str) -> mx.rnn.SequentialRNNCell:
+class SequentialRNNCellParallelInput(mx.rnn.SequentialRNNCell):
+    """
+    A SequentialRNNCell, where an additional "parallel" input can be given at
+    call time and it will be added to the input of each layer
+    """
+    def __init__(self, params=None):
+        super().__init__(params)
+    
+    def __call__(self, inputs, parallel_inputs, states):
+        # Adapted copy of mx.rnn.SequentialRNNCell.__call__()
+        self._counter += 1
+        next_states = []
+        pos = 0
+        for cell in self._cells:
+            assert not isinstance(cell, mx.rnn.BidirectionalCell)
+            length = len(cell.state_info)
+            state = states[pos:pos+length]
+            pos += length
+            concat_inputs = mx.sym.concat(inputs, parallel_inputs)
```

Comment:
on what axis does this concatenate? What are the shapes of inputs / parallel_inputs?